client/src/utils/systems.js

```diff
@@ -63,11 +63,12 @@ export function findSystemOrProjectDisplayName(
   scheme,
   systemList,
   projectsList,
-  system
+  system,
+  projectTitle
```

Comment:
See above comment - since projectTitle is now a requirement of findProjectTitle, this function now exhibits more than one layer of passing projectTitle.

Reply:
Hmm, projectTitle is not a requirement of findProjectTitle - it's the fallback if no matching project is found in the project list

Reply:
so the fallback is null if not specified?

Reply:
if the project id from the current project `system` state is not found in the `projectsList`, then it looks for the `projectTitle` from the requested project metadata. If that's also not found, then yes it'll be a blank string since there's nowhere else to look

Reply:
It looks for the project in the loaded projectsList state first, since that is faster to render than waiting fo the project metadata to load